code/game/objects/effects/portals.dm

```diff
@@ -22,7 +22,7 @@
 	set waitfor = FALSE
 	src.teleport(user)
 
-/obj/effect/portal/New(loc, turf/target, creator=null, lifespan=300)
+/obj/effect/portal/New(loc, turf/target, creator=null, lifespan=rand(300,600))

```

Comment:
```suggestion
/obj/effect/portal/New(loc, turf/target, creator=null, lifespan=rand(240,480))
```
Only because it both being 100% guaranteed and having the possiblity of doubling its time seems like a large buff. This introduces a small chance of less time and a large chance more time, without potentially doubling it. 

It may not be as large of a buff as I perceive it as, or telescience may need the buff, in which case feel free to ignore!

Reply:
It's already like... 5 minutes, It being 4 minutes to 8 minutes instead of 5 minutes to 10 minutes is fine, IMO. @ParadoxSpace may want to think about implementing this to placate Alberyk

Reply:
genius